src/features/import/utils/prepareImportOperation.ts

```diff
@@ -0,0 +1,63 @@
+import { CellData, ColumnKind, Sheet } from './types';
```

Comment:
The name of this file `prepareImportOperation` (singular) does not match the name of the default exported function `prepareImportOperations` (plural, which is correct).